tutor/src/components/book-content-mixin.cjsx

```diff
@@ -25,12 +25,14 @@ IS_INTRO_SELECTORS = '.splash img, [data-type="cnx.flag.introduction"]'
 
 LinkContentMixin =
   componentDidMount:  ->
+    @_isMounted = true
```

Comment:
_Minor:_ Since this is a mixin, would these fields accidentally trample the same field in `resize-listener-mixin`? Just wondering if the field name should be prefixed or not. 

But I'm just mentioning it as a minor issue; the meaning of `_isMounted` is pretty clear to me